packages/amplify_auth_cognito/test/amplify_auth_cognito_signup_test.dart

```diff
@@ -64,7 +65,7 @@ void main() {
             username: 'testUser',
             password: '123',
             options: CognitoSignUpOptions(userAttributes: {
-              "email": "test@test.com",
+              CognitoUserAttributes.email: "test@test.com",
```

Comment:
Nit: Kinda seems like we could just extract and use the same request three times? Ditto with some of the other tests.

Reply:
Probably, yeah. It would be good to do a sweep through all the tests at some point and clean them up.